### ISSUE TITLE
" operator crashes Edwood

### ISSUE THREAD

**Issue by rjkroege** [rjkroege/edwood#43]
I created a `win` session. I typed some commands. I used the `"` shell command by itself. (Which shouldn't have worked properly but never mind that.) Edwood promptly crashed:

```
panic: runtime error: slice bounds out of range
goroutine 50 [running]:
main.xfidutfread(0xc420394000, 0xc420382128, 0xc40, 0xb)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:901 +0x3f8
main.xfidread(0xc420394000)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:332 +0x30a
main.xfidctl(0xc420394000, 0xc4200b60c0)
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/xfid.go:46 +0x87
created by main.xfidallocthread
        /Users/rjkroege/tools/gopkg/src/github.com/rjkroege/edwood/acme.go:628 +0x1e5
```

**Comment by paul-lalonde** [comment]
Does it repro?  I'm not getting it.  I suspect the code path that inserts > RBUFSIZE elements from fsys.

**Comment by paul-lalonde** [comment]
Yes, there's unicode length-handling bug in that path.  Fixing.

**Comment by paul-lalonde** [comment]
Fixed in a9850d465cdd8736b3312953e10e18a0c9096936
